src/libs/Navigation/linkingConfig/config.ts

```diff
@@ -259,6 +259,12 @@ const config: LinkingOptions<RootStackParamList>['config'] = {
                         [SCREENS.WORKSPACE.INVITE_MESSAGE]: {
                             path: ROUTES.WORKSPACE_INVITE_MESSAGE.route,
                         },
+                        [SCREENS.WORKSPACE.CATEGORY_SETTINGS]: {
+                            path: ROUTES.WORKSPACE_CATEGORY_SETTINGS.route,
+                            parse: {
+                                categoryName: (categoryName: string) => decodeURI(categoryName),
```

Comment:
Regarding to **checklist**, It appears that this is a new pattern. Previously, parsing and linking were not utilized in this manner, In my opinion, it would be more effective to decode the category name before passing it to the route props.